### PR TITLE
Datahub message not available

### PR DESCRIPTION
**Pull request by GusBaamonde**
- Add the message to users who have not enabled the on-site behavior function in Spanish and English.
- Add the message to users who have not added a domain in both languages in Spanish and English.
- 

message-datahub

### CODE REVIEW

File: src/i18n/en.json [hunk 1]
```diff
@@ -35,8 +35,8 @@
     "you_want_create_account": "Do not you have an account yet?"
   },
   "reports": {
-    "datahub_not_active_HTML": "This feature is not enabled, please check <a href=\"{dopplerBaseUrl}/ControlPanel/CampaignsPreferences/SiteTrackingSettings\">On-Site Tracking preferences</a>.",
-    "no_domains_HTML": "There are not configured domains, please check <a href=\"{dopplerBaseUrl}/ControlPanel/CampaignsPreferences/SiteTrackingSettings\">On-Site Tracking preferences</a>."
+    "datahub_not_active_HTML": "<h4>Track user behavior and optimize your Marketing actions</h4><p>How do users behave on your Website or E-commerce? Track their behavior for a period of time and find out which are the most visited pages, how many of those visits have an Email identified by Doppler and how many don’t. By tracking the user’s journey you'll be able to detect vanishing points and opportunities for improvement!</p><p>You haven’t enabled On-Site Tracking yet. You can trigger it from the <a href=\"{dopplerBaseUrl}/ControlPanel/CampaignsPreferences/SiteTrackingSettings\">On-Site Tracking</a> option in the Control Panel.</p>",
```
Comment (cbernat) [inline src/i18n/en.json]: I'm not so fond of having so much html inside resources file. But we can change it later.

File: src/i18n/en.json [hunk 2]
```diff
@@ -35,8 +35,10 @@
...
   },
   "reports": {
-    "datahub_not_active_HTML": "This feature is not enabled, please check <a href=\"{dopplerBaseUrl}/ControlPanel/CampaignsPreferences/SiteTrackingSettings\">On-Site Tracking preferences</a>.",
-    "no_domains_HTML": "There are not configured domains, please check <a href=\"{dopplerBaseUrl}/ControlPanel/CampaignsPreferences/SiteTrackingSettings\">On-Site Tracking preferences</a>."
+    "datahub_not_active_title":"Track user behavior and optimize your Marketing actions",
+    "datahub_not_active_HTML": "<p>How do users behave on your Website or E-commerce? Track their behavior for a period of time and find out which are the most visited pages, how many of those visits have an Email identified by Doppler and how many don’t. By tracking the user’s journey you'll be able to detect vanishing points and opportunities for improvement!</p><p>You haven’t enabled On-Site Tracking yet. You can trigger it from the <a href=\"{dopplerBaseUrl}/ControlPanel/CampaignsPreferences/SiteTrackingSettings\">On-Site Tracking</a> option in the Control Panel.</p>",
+    "datahub_not_active_title":"Track user behavior and optimize your Marketing actions",
```
Comment (andresmoschini) [inline src/i18n/en.json]:
```suggestion
    "no_domains_title":"Track user behavior and optimize your Marketing actions",
```

Right?

File: src/i18n/en.json [hunk 3]
```diff
@@ -35,8 +35,10 @@
     "you_want_create_account": "Do not you have an account yet?"
   },
   "reports": {
-    "datahub_not_active_HTML": "This feature is not enabled, please check <a href=\"{dopplerBaseUrl}/ControlPanel/CampaignsPreferences/SiteTrackingSettings\">On-Site Tracking preferences</a>.",
-    "no_domains_HTML": "There are not configured domains, please check <a href=\"{dopplerBaseUrl}/ControlPanel/CampaignsPreferences/SiteTrackingSettings\">On-Site Tracking preferences</a>."
+    "datahub_not_active_title":"Track user behavior and optimize your Marketing actions",
```
Comment (andresmoschini) [inline src/i18n/en.json]: Please, sort these entries alphabetically:

```
datahub_not_active_HTML
datahub_not_active_title
no_domains_HTML
no_domains_title
```

**Comment by andresmoschini** [comment]
🚀

**Pull request merged**